### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by Lucas-obrien**
@sachhm
Please review!

### CODE REVIEW

File: prac_06/guitar.py [hunk 1]
```diff
@@ -4,20 +4,18 @@
 class Guitar:
     """Construct a guitar instance"""
 
-    def __init__(self, name="", year=0, cost=0):
+    def __init__(self, name="", year=0, cost=0.0):
```
Comment (sachhm) [inline prac_06/guitar.py]: Do we need to initialize as 0.0?  
image

File: prac_06/car.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        """
+        if distance > self.fuel:
+            distance = self.fuel
+            self.fuel = 0
+        else:
+            self.fuel -= distance
+        self._odometer += distance
+        return distance
+
+    def __str__(self):
```
Comment (sachhm) [inline prac_06/car.py:34]: Missing docstring.

File: prac_06/programming_language.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def is_dynamic(self):
+        """Return True or False if programming language is dynamically typed"""
+        return self.typing == "Dynamic"
+
+    def __str__(self):
```
Comment (sachhm) [inline prac_06/programming_language.py:23]: Missing docstring

**Comment by sachhm** [comment]
Your code looks good. Just a bit of nitpicking on my part 😆, and just a couple of docstrings here and there.

File: prac_06/guitar.py (hunk 1 above)
Comment (Lucas-obrien) [reply prac_06/guitar.py]: I was thinking there would be issues with changing data types in the class, but apparently not!
Will change!

**Comment by Lucas-obrien** [comment]
@sachhm All updated, how does it look?

**Comment by sachhm** [comment]
Looks solid. Again, these were fairly minor issues so nothing too much to change, but good job! 😸

**Pull request merged**